src/components/OrganizationAccordion.tsx

```diff
@@ -0,0 +1,116 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
```

Comment:
This component is almost an exact copy of what was inside OrganizationHierarchyFilter earlier. The only changes are a few variable name changes, two props added to allow different displays in the two use cases and one reorganization that maintains the logic. OrganizationHierarchyFilter is now using this component as well.